packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx

```diff
@@ -4,6 +4,8 @@ import CFDInstrumentsLabelHighlighted from './cfd-instruments-label-highlighted'
 import CFDCompareAccountsDescription from './cfd-compare-accounts-description';
 import CFDCompareAccountsTitleIcon from './cfd-compare-accounts-title-icon';
 import CFDCompareAccountsPlatformLabel from './cfd-compare-accounts-platform-label';
+import { Text } from '@deriv/components';
```

Comment:
Put the '@deriv/components' imports on top of the the other imports.
The order should be: 1. external libraries 2. deriv packages 3. internal components